dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Basics/CombiTimeTable.mo

```diff
@@ -0,0 +1,41 @@
+within Dynawo.Electrical.Controls.Basics;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite
+* of simulation tools for power systems.
+*/
+
+model CombiTimeTable "Table look-up with respect to time and linear/periodic extrapolation methods (data from a two-column table in a .txt file) with one output"
+  import Modelica;
+  import Dynawo;
+
+  parameter Modelica.Blocks.Types.Extrapolation Extrapolation
```

Comment:
Is it useful ? Does it work fine in dynawo ?

Reply:
It allows the user to choose among several options : HoldLastPoint, LastTwoPoint, Periodic, NoExtrapolation. It works well under Dynawo, I have tested all four options.